ui/components/app/signature-request/signature-request-data/signature-request-data.test.js

```diff
@@ -59,410 +60,251 @@ describe('Signature Request Data', () => {
       },
     };
 
-    let messageData;
     const store = configureMockStore()(mockStore);
 
-    beforeEach(() => {
-      messageData = {
-        domain: {
-          chainId: 97,
-          name: 'Ether Mail',
-          verifyingContract: '0xCcCCccccCCCCcCCCCCCcCcCccCcCCCcCcccccccC',
-          version: '1',
+    const rawMessage = {
+      domain: {
+        chainId: 97,
+        name: 'Ether Mail',
+        verifyingContract: '0xCcCCccccCCCCcCCCCCCcCcCccCcCCCcCcccccccC',
+        version: '1',
+      },
+      message: {
+        contents: 'Hello, Bob!',
+        from: {
+          name: 'Cow',
+          wallets: [
+            '0xCD2a3d9F938E13CD947Ec05AbC7FE734Df8DD826',
+            '0xDeaDbeefdEAdbeefdEadbEEFdeadbeEFdEaDbeeF',
+            '0x06195827297c7A80a443b6894d3BDB8824b43896',
+          ],
         },
-        message: {
-          contents: 'Hello, Bob!',
-          from: {
-            name: 'Cow',
+        to: [
+          {
+            name: 'Bob',
             wallets: [
-              '0xCD2a3d9F938E13CD947Ec05AbC7FE734Df8DD826',
-              '0xDeaDbeefdEAdbeefdEadbEEFdeadbeEFdEaDbeeF',
-              '0x06195827297c7A80a443b6894d3BDB8824b43896',
+              '0xbBbBBBBbbBBBbbbBbbBbbbbBBbBbbbbBbBbbBBbB',
+              '0xB0BdaBea57B0BDABeA57b0bdABEA57b0BDabEa57',
+              '0xB0B0b0b0b0b0B000000000000000000000000000',
             ],
           },
-          to: [
-            {
-              name: 'Bob',
-              wallets: [
-                '0xbBbBBBBbbBBBbbbBbbBbbbbBBbBbbbbBbBbbBBbB',
-                '0xB0BdaBea57B0BDABeA57b0bdABEA57b0BDabEa57',
-                '0xB0B0b0b0b0b0B000000000000000000000000000',
-              ],
-            },
-          ],
-        },
-        primaryType: 'Mail',
-        types: {
-          EIP712Domain: [
-            { name: 'name', type: 'string' },
-            { name: 'version', type: 'string' },
-            { name: 'chainId', type: 'uint256' },
-            { name: 'verifyingContract', type: 'address' },
-          ],
-          Mail: [
-            { name: 'from', type: 'Person' },
-            { name: 'to', type: 'Person[]' },
-            { name: 'contents', type: 'string' },
-          ],
-          Person: [
-            { name: 'name', type: 'string' },
-            { name: 'wallets', type: 'address[]' },
-          ],
-        },
-      };
-    });
-
```

Comment:
Hey are test cases below no longer relevant ?

Reply:
They were seemingly misunderstandings of what data the component should show, so I removed them because they would fail now that I changed the component input to be valid too 😅